packages/core/components/Drawer/styles.module.css

```diff
@@ -20,7 +20,6 @@
   font-size: var(--puck-font-size-xxs);
   justify-content: space-between;
   align-items: center;
-  cursor: grab;
   transition: background-color 50ms ease-in, color 50ms ease-in;
 }
 
```

Comment:
Happy to do this as a follow-up, but we should probably add a disabled state here, to give the user a visual indication this is not draggable

```css
.DrawItem--disabled {
  background: var(--puck-color-grey-11);
  color: var(--puck-color-grey-05);
  cursor: not-allowed; /** Move this out of inline styles */
}
```

And make sure the hover state doesn't occur when disabled

```css
.Drawer:not(.Drawer--isDraggingFrom) .DrawerItem-draggable:hover:not(.DrawItem--disabled) {
```

<img width="311" alt="image" src="https://github.com/user-attachments/assets/517e135e-b55b-4126-a2b7-d2db7b7a162b">